src/main/java/module-info.java

```diff
@@ -20,6 +20,10 @@
     requires com.pi4j.plugin.gpiod;
    // requires com.pi4j.library.kernel;
     //requires com.pi4j.plugin.rp1spi;
+
+    requires pi4j.drivers;  // Note drivers vs. driver ¯\_(ツ)_/¯
+    uses com.pi4j.driver.sensor.bmx280.Bmx280Driver;
```

Comment:
Shouldn't this be fixed in pi4j-drivers in that the package is plural?

Reply:
I am not sure. Ideally, they all would be consistent -- I don't really mind which way... 
Let's sort this out in slack before we pull this in...